charts/velero/templates/deployment.yaml

```diff
@@ -118,6 +118,13 @@ spec:
               value: {{ default "none" $value }}
           {{- end }}
           {{- end }}
+          {{- range .Values.environmentSecrets }}
```

Comment:
I don't think this is going to work. Velero only works with 1 secret at the time. If I'm misunderstanding, maybe please explain what this is trying to solve, thank you,